### PR TITLE
Branch java docs

### LINKED ISSUE #362: Add JavaDocs

### PR DESCRIPTION
**Pull request by durianpancakes**

### CODE REVIEW

File: src/main/java/seedu/duke/model/event/tuition/EventTuition.java [hunk 1]
```diff
@@ -7,25 +7,47 @@
 import java.util.Calendar;
 
 //@@author durianpancakes
+
```
Comment (AndreWongZH) [inline src/main/java/seedu/duke/model/event/tuition/EventTuition.java]: I think there shouldnt be spaces between the authorship and the javadoc

File: src/main/java/seedu/duke/ui/CalendarWeekRendererUtils.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package seedu.duke.ui;
 
+import com.sun.jdi.ThreadReference;
```
Comment (AndreWongZH) [inline src/main/java/seedu/duke/ui/CalendarWeekRendererUtils.java]: Is this import intended? Might be accidentally added in.

File: src/main/java/seedu/duke/ui/CalendarWeekRendererUtils.java [hunk 3]
```diff
@@ -125,15 +208,23 @@ public String getDescriptions(ArrayList<Event> events, int counterIndex) {
         return descriptionsString.toString();
     }
 
+    /**
+     * Obtain the Event's start end time to be rendered.
```
Comment (AndreWongZH) [inline src/main/java/seedu/duke/ui/CalendarWeekRendererUtils.java]: The first word should start with an -s. Maybe use Obtains instead?

**Review by AndreWongZH — COMMENTED** [review]
Looks great. Just some minor changes.

File: src/main/java/seedu/duke/ui/UserInterface.java [hunk 4]
```diff
@@ -50,10 +69,19 @@ public String getUserCommand() {
         return in.nextLine();
     }
 
+    /**
+     * Helper function to show an empty line.
```
Comment (elizabethcwt) [inline src/main/java/seedu/duke/ui/UserInterface.java]: Perhaps javadocs wouldn't be necessary for self-explanatory methods like this?

**Review by untitle4 — COMMENTED** [review]
The descriptions look clear enough.

**Comment by durianpancakes** [comment]
Resolves #362

**Pull request merged**